### PR TITLE
Add multisite chapter to the Laravel Valet page

### PR DESCRIPTION
**Pull request by gabriel-glo**
Sub-domain install is explained in more detail since it is more common in our projects and Valet is already configured for it.

For sub-directory install I only provided instructions on how to install custom Valet drivers and the rest is pretty much the same.

### CODE REVIEW

**Review by iruzevic — APPROVED** [review]

**Review by IvanGrginovInf — APPROVED** [review]

File: Development environment/Laravel Valet setup.md [hunk 1]
```diff
@@ -62,6 +62,76 @@ $ wp core download
...
+
+This type of multisite allows you to access sites with a sub-domain. The URLs will look like this:
+
+```bash
+dev.multisite-sub.test               # Main site
+site-1.dev.multisite-sub.test
+site-2.dev.multisite-sub.test
+```
+
+This type of multisite is more common and ususally requested by the clients. It is also a bit more complicated to set up. To achive this some changes to the server configuration are required bofore the multisite can be installed and primarily you need to point multiple domains/subdomains to the same directory. Luckly, Valet is already prepared for that and has all the configurations for this type of multisite.
```
Comment (iobrado) [inline Development environment/Laravel Valet setup.md:81]:
```suggestion
This type of multisite is more common and usually requested by the clients. It is also a bit more complicated to set up. To achieve this some changes to the server configuration are required before the multisite can be installed and primarily you need to point multiple domains/subdomains to the same directory. Luckily, Valet is already prepared for that and has all the configurations for this type of multisite.
```

File: Development environment/Laravel Valet setup.md [hunk 2]
```diff
@@ -62,6 +62,76 @@ $ wp core download
...
+2/6 [--title=<title>]: Site Title
+3/6 [--email=<email>]: admin@mail.com  # This will add existing or create a new Administrator user for this site
+4/6 [--network_id=<network-id>]: 1     # Network to associate new site with
+5/6 [--private] (Y/n): n
+6/6 [--porcelain] (Y/n):
+```
+
+### Multisite with sub-directories
+
+This type of multisite allows you to access sites with a sub-directories. The URLs will look like this:
```
Comment (goranalkovic-infinum) [inline Development environment/Laravel Valet setup.md:116]:
```suggestion
This type of multisite allows you to access sites with sub-directories. The URLs will look like this:
```

**Review by goranalkovic-infinum — DISMISSED** [review]

**Review by iobrado — DISMISSED** [review]

File: Development environment/Laravel Valet setup.md [hunk 3]
```diff
@@ -62,6 +62,76 @@ $ wp core download
...
+
+This type of multisite allows you to access sites with a sub-domain. The URLs will look like this:
+
+```bash
+dev.multisite-sub.test               # Main site
+site-1.dev.multisite-sub.test
+site-2.dev.multisite-sub.test
+```
+
+This type of multisite is more common and usually requested by the clients. It is also a bit more complicated to set up. To achieve this some changes to the server configuration are required before the multisite can be installed and primarily you need to point multiple domains/subdomains to the same directory. Luckily, Valet is already prepared for that and has all the configurations for this type of multisite.
```
Comment (dingo-d) [inline Development environment/Laravel Valet setup.md:81]:
```suggestion
This type of multisite is more common and usually requested by the clients. It is also a bit more complicated to set up. To achieve this some changes to the server configuration are required before the multisite can be installed. You need to point many domains/subdomains to the same directory. Luckily, Valet is already prepared for that and has all the configurations for this type of multisite.
```

File: Development environment/Laravel Valet setup.md [hunk 4]
```diff
@@ -62,6 +62,76 @@ $ wp core download
...
+
+Taking the above URLs as an example we can start installing the multisite. First create a main domain for the site by navigating to the directory and running: `valet link dev.multisite-sub`.
```
Comment (dingo-d) [inline Development environment/Laravel Valet setup.md:83]:
```suggestion
Taking the above URLs as an example we can start installing the multisite. First, create a main domain for the site by navigating to the directory and running: `valet link dev.multisite-sub`.
```

File: Development environment/Laravel Valet setup.md [hunk 5]
```diff
@@ -62,6 +62,76 @@ $ wp core download
...
+
+After that add additional domains with: `valet link site-1.dev.multisite-sub`.
```
Comment (dingo-d) [inline Development environment/Laravel Valet setup.md:85]:
```suggestion
After that, add additional domains with: `valet link site-1.dev.multisite-sub`.
```

File: Development environment/Laravel Valet setup.md [hunk 6]
```diff
@@ -62,6 +62,76 @@ $ wp core download
...
+This type of multisite allows you to access sites with sub-directories. The URLs will look like this:
+
+```bash
+dev.multisite-sub.test               # Main site
+dev.multisite-sub.test/site-1
+dev.multisite-sub.test/site-2
+```
+
+Laravel Valet does not come with configuration for this setup out of the box. For this you will need to use a custom driver.
+Guide for the installation of the custom drivers can be found [here](https://laravel.com/docs/5.6/valet#custom-valet-drivers).
```
Comment (dingo-d) [inline Development environment/Laravel Valet setup.md:125]:
```suggestion
Guide for the installation of the custom drivers can be found [here](https://laravel.com/docs/8.x/valet#custom-valet-drivers).
```

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left some suggestions

**Review by kmaric69 — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]

**Pull request merged**